### DISCUSSION TITLE
Subject: Account Flagged Due to Deceptive Content

### DISCUSSION THREAD

**Discussion by Ben22-Developer** [community/community#146187] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello fellows,
I'm writing to seek your assistance regarding an issue I've encountered with my GitHub account. Recently, I created a web design clone of Facebook for a user login page. After it was reported as dangerous, I removed it from online. However, since then, all the sites associated with my GitHub account have been flagged as dangerous.
Here are the details:
• GitHub Username: [@Ben22-Dev]
• Steps Taken: I have removed the reported content from my GitHub account and ensured no deceptive or harmful content remains.
• Problem: Despite these actions, all my other projects and sites on GitHub are still being flagged as dangerous.
• Request: Could you please assist me on this issue ?
Thank you for your time and assistance.
Best regards, [@Ben22-Dev]

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Ben22-Developer, 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
